src/main/webapp/creatives/maps/simid_map_creative.test.js

```diff
@@ -1,12 +1,21 @@
 /** Tests to see if Find Nearest X button is appending correct text */
-
 import SimidMapCreative from './simid_map_creative.js';
 
-let testMap;
+let testMap, googleMaps;
```

Comment:
Give each of these their own line
https://google.github.io/styleguide/jsguide.html#features-one-variable-per-declaration